### DISCUSSION TITLE
Cases when transaction resolver resolves In/Out Transaction

### DISCUSSION THREAD

**Discussion by topcoderasdf** [eprbell/dali-rp2#93] [Q&A] [answered]
I've noticed that transaction resolver can resolve in/out or out/in transaction. Would there be cases where users should be aware and use in/out instead of intra transaction?

**Comment by eprbell** [comment, marked as answer]
Not sure I fully understand the question. The transaction resolver can match and join 2 incomplete intra transactions or 2 incomplete out/in transactions. For more details on this read: https://github.com/eprbell/dali-rp2/blob/main/docs/configuration_file.md#partial-transactions-and-transaction-resolution
Hope this helps.

**Reply by eprbell** [reply]
@topcoderasdf, does the above answer your question?

**Reply by topcoderasdf** [reply]
Sorry for the late reply. Totally forgot about this. I got confused because I forgot about transaction hints that could be used to resolve various transaction types (including in/out). I only thought intra/intra could resolved as I was going through the Coinbase plugin, so initially I couldn't understand the use cases for in/out or out/in transaction resolver.
